about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Body Bulding</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+  <script src="https://kit.fontawesome.com/c6f38472f5.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header>
+    <div class="wrapper flex">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My Page</a>
+      <a href="#">Logout</a>
+    </div>
+  </header>
+
+  <i class="fa-sharp fa-solid fa-bars menu"></i>
+  <div class="navigation">
+    <i class="fa-solid fa-xmark xmark"></i>
+    <nav class="wrapper">
+      <div class="nav-top">
+        <a href="index.html">
+          <img class="logo" src="assets/images/logo2.png" alt="logo">
+          <p class="max-font-26"><span>SP Fitness</span><br>center 2023</p>
+        </a>
+      </div>
+      <ul class="nav-links flex column">
+        <li><a class="nav-link max-font-26" href="about.html">About</a></li>
+        <li><a class="nav-link max-font-26" href="#">Program</a></li>
+        <li><a class="nav-link max-font-26" href="#">Join</a></li>
+        <li><a class="nav-link max-font-26" href="#">Sponsor</a></li>
+        <li><a class="nav-link max-font-26 " href="#">News</a></li>
+      </ul>
+      <div class="nav-bottom flex">
+        <button class="max-font-26" type="button">SPFC Story</button>
+      </div>
+    </nav>
+  </div>
+  <main>
+    <section id="headline" class="section">
+      <div class="greeting">
+        <p>"Hello! Champion"</p>
+      </div>
+      <div class="wrapper">
+        <h1>Super Power Fitness Center 2023</h1>
+        <p class="description">Super Power Fitness Center is a premier gym offering various services and facilities to
+          help you achieve your fitness goals. Whether you want to lose weight, build muscle, improve your health or
+          just
+          have fun, we have something for you. Join us today and discover the benefits of working out with our qualified
+          trainers, state-of-the-art equipment and supportive community.</p>
+        
+        <p class="contact">Please contact us per Email for any further questions about SP Fitness Center 2023!</p>
```

Comment:
- [ ] I think it would be great here if you make the background the color white like the given design since you made it white in the desktop version but in the mobile version you just added a border but I highly recommend to change the background color for the paragraph as well ✨🚀🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/235009245-44d1c250-af4a-4060-ba89-bdc26d6344a4.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235009298-b55c9fb2-9853-4cf7-b10a-4ab6ee517325.png)